Makefile

```
@@ -35,6 +35,9 @@ uninstall:
 test:
 	go test ./...
 
+lint:
```

```suggestion
.PHONY: lint
lint:
```

the same would apply for other targets. The meaning is that "lint" is not a file that make should check for existance but this target is a "virtual" target that always needs to be executed on call. Minor optimization.